gateway/mw_request_signing_test.go

```diff
@@ -436,31 +436,41 @@ func TestAPISpec_StripListenPath(t *testing.T) {
 	})
 
 	t.Run("On", func(t *testing.T) {
-		ts.Gw.BuildAndLoadAPI(func(spec *APISpec) {
```

Comment:
The gateway created in the tests was shared between all tests, but modified in test